README.md

```diff
@@ -1 +1,33 @@
-# Front-End-Internship-Task
\ No newline at end of file
+# Front-End-Internship-Task
+
+This is a front-end task for your internship in Forte Digital. This repo is created based on `create-react-app`. It consists of a list of interns. It has two pages:
+​
+1. List of interns
+1. Edit intern page
+
+To check your front-end skills please complete two tasks specified in this document and send us zipped repo to an address you have received in email. Before zipping repo remember to remove `node_modules` directory.
```

Comment:
detail but, `the email` you can check is the text correct with https://app.grammarly.com/